src/v0/destinations/facebook_pixel/utils.js

```diff
@@ -192,10 +184,6 @@ const transformedPayloadData = (
         delete clonedCustomData[eventProp];
       }
     }
-    const isCustomProperty = customEventProperties[eventProp] || false;
-    if (isStandard && !isCustomProperty && !isDefaultPiiProperty) {
-      delete clonedCustomData[eventProp];
-    }
```

Comment:
Could you please let me know why you're removing this entirely? It does not seem the same with device mode implementation.

Reply:
In line 195 we are checking if the current property being iterated is a mapped property for standard event. (This mapping is removed hence we are removing this. )
Refer removed line 173

> `finalEventCustomProperties.forEach((property) => {
>     const singularConfigInstance = property;
>     customEventProperties[singularConfigInstance.eventCustomProperties] = true;
>   });`

Now in the next check, we were deleting this custom property if the event was a standard event, the property is not mapped in the now removed mapping and the property is not a defaultPiiProperty. SInce [FB supports passing any custom property for standard events](https://developers.facebook.com/docs/marketing-api/conversions-api/parameters/custom-data#custom-properties), removing this condition